schema/interface.ipv4.yml

```diff
@@ -168,6 +196,13 @@ properties:
       examples:
       - 8.8.8.8
       - 4.4.4.4
+  dhcp-snoop-vlan-enable:
+    description: "Enables DHCP Snooping on a VLAN"
+    type: boolean
+    default: false
+    enum: 
```

Comment:
boolean is only true or false, it doesn't need enumerations.

Reply:
Understood. will remove from everywhere i added.